JSDemos/Demos/Toolbar/Multiline/React/App.js

```diff
@@ -0,0 +1,323 @@
+import React from 'react';
+import Toolbar, { Item } from 'devextreme-react/toolbar';
+import { Button } from 'devextreme-react/button';
+import { ButtonGroup } from 'devextreme-react/button-group';
+import RadioGroup from 'devextreme-react/radio-group';
+import Resizable from 'devextreme-react/resizable';
+import CheckBox from 'devextreme-react/check-box';
+import DateBox from 'devextreme-react/date-box';
+import DropDownButton from 'devextreme-react/drop-down-button';
+import SelectBox from 'devextreme-react/select-box';
+import notify from 'devextreme/ui/notify';
+import 'devextreme/ui/select_box';
+import {
+  fontSizes,
+  lineHeights,
+  fontFamilies,
+  fontStyles,
+  textAlignItems,
+  textAlignItemsExtended,
+  listTypes,
+} from './data.js';
+
+const lineHeightDefault = lineHeights[1].lineHeight;
+const textAlignDefault = [textAlignItems[0].alignment];
+const fontSizeDefault = fontSizes[2].size;
+
+const toolbarLineModes = [
+  {
+    text: 'Multiline mode',
+    value: true,
+  },
+  {
+    text: 'Singleline mode',
+    value: false,
+  },
+];
+
+const attachButtonOptions = {
+  icon: 'attach',
+  text: 'Attach',
+  onClick: () => notify('Attach button has been clicked!'),
+};
+
+const addButtonOptions = {
+  icon: 'add',
+  text: 'Add',
+  onClick: () => notify('Add button has been clicked!'),
+};
+
+const removeButtonOptions = {
+  icon: 'trash',
+  text: 'Remove',
+  onClick: () => notify('Remove button has been clicked!'),
+};
+
+const aboutButtonOptions = {
+  icon: 'help',
+  text: 'About',
+  onClick: () => notify('About button has been clicked!'),
+};
+
+function App() {
+  const [lineHeight, setLineHeight] = React.useState(lineHeightDefault);
```

Comment:
It's a minor comment
But, in my point of view, it will be cool to import React hooks directly.
I mean something like this:

```
import React, {useCallback, useState}  from 'react';

function MyComponent() => {
  const [state,setState] = useState(true);
}
```

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/b97c28826b4feb1b12e941e53837224b4c3838fb

Reply:
`TypeError: useState is not a function` in [this job](https://github.com/DevExpress/devextreme-demos/actions/runs/3119218375/jobs/5059134305) before this commit.